src/platform/cannonlake/include/platform/platform.h

```diff
@@ -118,6 +118,9 @@ struct sof;
 /* minimal L1 exit time in cycles */
 #define PLATFORM_FORCE_L1_EXIT_TIME	482
 
+/* the half of SSP port fifo depth */
+#define PLATFORM_SSP_FIFO_DEPTH_HALF	8
```

Comment:
Best to define a macro for FIFO depth and you can then divide it by 2 where you need to know half depth.

Reply:
ok. i will update it.